### ISSUE TITLE
Landing page links point to wrong address

### ISSUE THREAD

**Issue by jsaathof** [LycheeOrg/Lychee#1281]
### Detailed description of the problem [REQUIRED]
I have the landing page enabled while running a docker install behind a reverse proxy (Apache2). The links (my name and "Gallery") point to the docker host which is not publicly accessible. When opening the gallery by hand on the proxy host the gallery works as expected. I have disabled the landing page to work around this issue.

### Steps to reproduce the issue
1. Open lychee via the reverse proxy
2. Click on Gallery

**Screenshots**
*If applicable, add screenshots to help explain your problem.*

### Output of the diagnostics  [REQUIRED]

    Diagnostics
    -------
    Warning: Dropbox import not working. dropbox_key is empty.

    System Information
    --------------
    Lychee Version (git):            No git data found.
    DB Version:                      4.4.0
    
    composer install:                --no-dev
    APP_ENV:                         production
    APP_DEBUG:                       false
    
    System:                          Linux
    PHP Version:                     8.1
    PHP User agent:                  Lychee/4 (https://lycheeorg.github.io/)
    Max uploaded file size:          100M
    Max post size:                   100M
    Max execution time:              3600
    PostgreSQL Version:              PostgreSQL 12.9 on x86_64-suse-linux-gnu, compiled by gcc (SUSE Linux) 7.5.0, 64-bit
    
    Imagick:                         1
    Imagick Active:                  1
    Imagick Version:                 1691
    GD Version:                      2.3.0

    Config Information
    --------------
    version:                         040400
    check_for_updates:               1
    sorting_Photos_col:              taken_at
    sorting_Photos_order:            ASC
    sorting_Albums_col:              min_taken_at
    sorting_Albums_order:            DESC
    imagick:                         1
    skip_duplicates:                 0
    small_max_width:                 0
    small_max_height:                360
    medium_max_width:                1920
    medium_max_height:               1080
    lang:                            en
    layout:                          0
    image_overlay_type:              none
    default_license:                 none
    compression_quality:             90
    full_photo:                      1
    delete_imported:                 1
    Mod_Frame:                       0
    Mod_Frame_refresh:               30
    landing_page_enable:             0
    landing_owner:                   Jurriaan Saathof
    landing_title:                   Jurriaan Saathof
    landing_subtitle:                
    landing_facebook:                
    landing_flickr:                  
    landing_twitter:                 
    landing_instagram:               
    landing_youtube:                 
    landing_background:              uploads/big/12fb044cf99851660e10c6edcacf4736.jpg
    thumb_2x:                        1
    small_2x:                        0
    medium_2x:                       0
    site_title:                      Xenophobia
    site_copyright_enable:           1
    site_copyright_begin:            2007
    site_copyright_end:              2022
    public_search:                   0
    additional_footer_text:          
    display_social_in_gallery:       0
    public_recent:                   0
    recent_age:                      1
    SL_enable:                       0
    SL_for_admin:                    0
    public_starred:                  0
    downloadable:                    0
    photos_wraparound:               1
    map_display:                     0
    zip64:                           1
    map_display_public:              0
    map_provider:                    Wikimedia
    force_32bit_ids:                 0
    map_include_subalbums:           0
    update_check_every_days:         3
    has_exiftool:                    1
    share_button_visible:            0
    has_ffmpeg:                      0
    import_via_symlink:              0
    location_decoding:               0
    location_decoding_timeout:       30
    location_show:                   1
    location_show_public:            0
    rss_enable:                      0
    rss_recent_days:                 7
    rss_max_items:                   100
    prefer_available_xmp_metadata:   0
    editor_enabled:                  1
    lossless_optimization:           0
    swipe_tolerance_x:               150
    swipe_tolerance_y:               250
    local_takestamp_video_formats:   .avi|.mov
    log_max_num_line:                1000
    unlock_password_photos_with_url_param: 0
    nsfw_visible:                    1
    nsfw_blur:                       0
    nsfw_warning:                    0
    nsfw_warning_admin:              0
    map_display_direction:           1
    album_subtitle_type:             oldstyle
    upload_processing_limit:         4
    public_photos_hidden:            1
    new_photos_notification:         0

### Browser and system
macOS Monterey 12.3.1, Firefox 99 / Safari 15.4

**Comment by d7415** [comment]
I'm going to close this as a duplicate of Lychee-Docker#89. Once #1217 is merged we'll have proper access to the trusted proxies config and can set it appropriately. Hopefully that won't take too long, but it is a substantial PR.

Until then you can try mounting a modified version of [this file](https://github.com/LycheeOrg/Lychee-Docker/issues/89#issuecomment-797836706) over the top as a workaround.
